defaults/main.yml

```diff
@@ -16,6 +16,13 @@ docker_py_version: "1.9.0"
 # 'latest' to install the newest version, null or empty string to install nothing.
 docker_compose_version: "1.8.0"
 
+# Extra python requirements.
+# There are times when we will need to be explicit about versions of other libraries
+# See README.md for detals.
+# Empty list means don't specify any additional versions
+docker_base_extra_python_requirements:
+  - "requests==2.7.0"
```

Comment:
this will cause the default deployment of the role to not install the proper requests version and require some additional configuration. we could just update as needed when we update the docker_compose and docker_py version...

i'm not sure that is the best approach, but i can make the change.